.github/workflows/ci.yml

```diff
@@ -78,9 +78,11 @@ jobs:
           native-image-job-reports: "true"
           cache: "maven"
       - name: "Install"
+        if: ${{ matrix.java != 'GraalVM' }}
         shell: bash
         run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
       - name: "Test"
+        if: ${{ matrix.java != 'GraalVM' }}
```

Comment:
Skipping these steps for GraalVM sounds good.

I'm realizing that there isn't a lot of overlap between the steps that we want to do for GraalVM and other JDKs--I think all of the steps here happen on other JDKs, or on GraalVM, but not both.

What do you think about adding a separate entry under `jobs:` for the GraalVM test, isntead of adding it to the matrix with the other JDKs and avoiding all of the checks for `java == graal` and `java != graal`?

Reply:
FTR: This was done in #1055. (I'll close this PR.)